starlark/log.go

```diff
@@ -46,5 +46,6 @@ func addDefaultLogger(t *starlark.Thread) {
 	loggerLocal := t.Local(identifiers.log)
 	if loggerLocal == nil {
 		t.SetLocal(identifiers.log, log.Default())
+		t.SetLocal(identifiers.logPath, defaults.logpath)
```

Comment:
If we are doing a log with a UUID in the filename, need to figure out the best way to pass this value around in the cmd until getting to the thread (when we can put it into the thread go context)